config/storybook/config.js

```diff
@@ -0,0 +1,18 @@
+import { configure, addDecorator } from '@storybook/react';
+import { withTests } from '@storybook/addon-jest';
+import results from '../../.jest-test-results.json';
+// automatically import all files ending in *.stories.js
+
+// addDecorator(
```

Comment:
Any reason to keep this around?

Reply:
Repurposed the decorator to wrap all stories with basic text styles that are normally applied by a consuming application.